jonelle-ramlal/06/exercise.test.js

```diff
@@ -0,0 +1,93 @@
+describe('06', () => {
+
+    // Tip: Use setTimeout to delay the execution of marking
+
+    const listOfPapers = [
+        {
+            subject: "Maths",
+            wasSubmitted: true,
+            markPaper: () => {
+                // add a promise here that resolves after 2 seconds
+                // and print "Maths paper marked"
+                return new Promise((resolve,reject) => {
+                    setTimeout(() => {
+                        resolve(console.log("Maths paper marked"))
```

Comment:
This works and you won't lose marks for it mainly because of how the instructions were given. 

But ideally, you would have wanted to resolve the text itself and not the console.log with the text.